src/main/scala/chisel3/tester/experimental/package.scala

```diff
@@ -3,9 +3,49 @@
 package chisel3.tester
 
 import chisel3._
+import chisel3.core.ActualDirection  // TODO needs to be a public API
+import chisel3.experimental.{DataMirror, MultiIOModule}
+import chisel3.tester.internal._
+import firrtl.ExecutionOptionsManager
 
+/** Your warranty is now void.
+  *
+  * experimental contains cutting edge features that are, well, experimental, and carry no
+  * expectation of long-term support. We may break experimental APIs at any time. These may not
+  * work as expected, or may have unforeseen side effects, or may be powerful yet dangerous.
+  *
+  * You have been warned.
+  */
 package object experimental {
-  implicit class uncheckedPokeableClock(x: Clock) {
+  type TesterOptions = chisel3.tester.internal.TesterOptions
+  val TesterOptions = chisel3.tester.internal.TesterOptions  // expose this internal object, whose "API" is unstable
 
+  implicit class ChiselScalatestOptionBuilder[T <: MultiIOModule](x: ChiselScalatestTester#TestBuilder[T]) {
```

Comment:
FYI, this forces us to `import chisel3.tester.experimental._` which may be good to avoid?

Reply:
It's now in its own sub package object, so you would instead either
```scala
import chisel3.tester.experimental.TestOptionBuilder._
```

```scala
import chisel3.tester.experimental.UncheckedClockPoke._
```

I don't think there's a way to do implicit conversions without a wildcard import.

Reply:
I've seen cases where the implicits are grouped in an `Implicits` object and then wildcard imported:
```scala
object foo {
  object Implicits {
   ...
  }
}

import ...foo.Implicits._
```